inc/class-wpseo-features.php

```diff
@@ -7,24 +7,34 @@
 
 /**
  * Class containing method for WPSEO Features.
+ *
+ * @deprecated 16.0
  */
 class WPSEO_Features {
 
 	/**
 	 * Checks if the premium constant exists to make sure if plugin is the premium one.
 	 *
+	 * @deprecated 16.0
+	 * @codeCoverageIgnore
+	 *
 	 * @return bool
 	 */
 	public function is_premium() {
-		return ( defined( 'WPSEO_PREMIUM_FILE' ) );
+		_deprecated_function( __FUNCTION__, '16.0', 'YoastSEO()->helpers->product->is_premium' );
+		return YoastSEO()->helpers->product->is_premium();
 	}
 
 	/**
 	 * Checks if using the free version of the plugin.
 	 *
+	 * @deprecated 16.0
+	 * @codeCoverageIgnore
+	 *
 	 * @return bool
 	 */
 	public function is_free() {
-		return ! $this->is_premium();
+		_deprecated_function( __FUNCTION__, '16.0', 'YoastSEO()->helpers->product->is_premium' );
+		return true;
```

Comment:
Is it intentional to always return true?

Reply:
Yes, as premium is now an addon that means free is always free. The logic of is_free = ! is_premium no longer applies as free and premium can both be active at the same time

Reply:
Thanks for the clarification!